app/services/refresh_reporting_views.rb

```diff
@@ -107,14 +107,17 @@ def refresh
         klass = name.constantize
         klass.refresh
       end
+      Statsd.instance.flush
     end
   end
 
   def benchmark_and_statsd(operation)
     name = "refresh_reporting_views.#{operation}"
     benchmark(name) do
       Datadog::Tracing.trace("refresh_matview", resource: operation) do |span|
-        yield
+        Statsd.instance.time(name) do
```

Comment:
We used to have statsd timing earlier which was removed in favour of tracing. However, the trace's time - [sample trace](https://app.datadoghq.com/apm/traces?query=%40_top_level%3A1%20env%3Aindia-production%20service%3Asimple_server%20operation_name%3Arefresh_matview&cols=service%2Cresource_name%2C%40duration%2C%40http.method%2C%40http.status_code%2C%40_duration.by_service&graphType=flamegraph&historicalData=true&saved-view-id=1277623&spanID=2209728970880325150&spanType=service-entry&spanViewType=metrics&timeHint=1666649615069&trace=AgAAAYQMDurdGcSDDQAAAAAAAAAYAAAAAEFZUU1Edm9BQUFDNGFicUVQVWlvM0ZRcgAAACQAAAAAMDE4NDBjNmEtYWJiNS00NjIxLWFlMzYtNDU3MmRkMzI1ZTQz&traceID=3122197871034418358&viz=stream&start=1666521747668&end=1666694547668&paused=false) can't be plotted on a graph over time.